apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/grpc/RetriableStargateBridge.java

```diff
@@ -18,33 +18,23 @@
 package io.stargate.sgv2.api.common.grpc;
 
 import io.grpc.StatusRuntimeException;
-import io.quarkus.grpc.GrpcClient;
-import io.quarkus.grpc.GrpcService;
 import io.smallrye.mutiny.Uni;
 import io.stargate.bridge.proto.QueryOuterClass;
 import io.stargate.bridge.proto.Schema;
 import io.stargate.bridge.proto.StargateBridge;
 import io.stargate.sgv2.api.common.config.GrpcConfig;
-import io.stargate.sgv2.api.common.grpc.qualifier.Retriable;
-import javax.inject.Inject;
-import javax.inject.Singleton;
 
 /**
  * An implementation of the {@link StargateBridge} that executes retries based on the
  * GrpcConfig.Retries configuration.
  */
-@Singleton
-@GrpcService
```

Comment:
This is the wrong annotation, that was exposing this as a service. However, I was confused by the following build time error that made me add it:

```
[ERROR] Failed to execute goal io.quarkus.platform:quarkus-maven-plugin:2.16.0.Final:build (default) on project sgv2-docsapi: Failed to build quarkus application: io.quarkus.builder.BuildException: Build failure: Build failed due to errors
[ERROR] 	[error]: Build step io.quarkus.arc.deployment.ArcProcessor#generateResources threw an exception: javax.enterprise.inject.spi.DeploymentException: java.lang.IllegalStateException: A gRPC service bean must be annotated with @io.quarkus.grpc.GrpcService: CLASS bean [types=[io.stargate.sgv2.api.common.grpc.RetriableStargateBridge, io.stargate.bridge.proto.StargateBridge, io.quarkus.grpc.MutinyService, java.lang.Object], qualifiers=[@Retriable, @Any], target=io.stargate.sgv2.api.common.grpc.RetriableStargateBridge]
[ERROR] 	at io.quarkus.arc.processor.BeanDeployment.processErrors(BeanDeployment.java:1223)
[ERROR] 	at io.quarkus.arc.processor.BeanProcessor.processValidationErrors(BeanProcessor.java:161)
[ERROR] 	at io.quarkus.arc.deployment.ArcProcessor.generateResources(ArcProcessor.java:559)
[ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
[ERROR] 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
[ERROR] 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[ERROR] 	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
[ERROR] 	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:909)
[ERROR] 	at io.quarkus.builder.BuildContext.run(BuildContext.java:281)
[ERROR] 	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
[ERROR] 	at java.base/java.lang.Thread.run(Thread.java:833)
[ERROR] 	at org.jboss.threads.JBossThread.run(JBossThread.java:501)
[ERROR] Caused by: java.lang.IllegalStateException: A gRPC service bean must be annotated with @io.quarkus.grpc.GrpcService: CLASS bean [types=[io.stargate.sgv2.api.common.grpc.RetriableStargateBridge, io.stargate.bridge.proto.StargateBridge, io.quarkus.grpc.MutinyService, java.lang.Object], qualifiers=[@Retriable, @Any], target=io.stargate.sgv2.api.common.grpc.RetriableStargateBridge]
[ERROR] 	at io.quarkus.grpc.deployment.GrpcServerProcessor.validateBindableService(GrpcServerProcessor.java:477)
[ERROR] 	at io.quarkus.grpc.deployment.GrpcServerProcessor.validateBindableServices(GrpcServerProcessor.java:465)
[ERROR] 	... 11 more
[ERROR] -> [Help 1]
[ERROR] 

```